fog/sql_recovery_db/src/lib.rs

```diff
@@ -2289,6 +2385,100 @@ mod tests {
         assert_eq!(tx_outs, records2);
     }
 
+    #[test_with_logger]
+    fn test_get_tx_outs_by_block_range_and_key(logger: Logger) {
+        let mut rng: StdRng = SeedableRng::from_seed([123u8; 32]);
```

Comment:
Optional/Equivalent: `StdRng::from_seed`
```suggestion
        let mut rng = StdRng::from_seed([123u8; 32]);
```

Reply:
lets stick with the convention in use in this file, we can change it later